pkg/ovs/openflow/ofctrl_action.go

```diff
@@ -94,9 +94,11 @@ func (a *ofCTAction) LoadToMark(value uint32) CTAction {
 	return a
 }
 
-func (a *ofCTAction) LoadToCtMark(mark *CtMark) CTAction {
-	field, _, _ := getFieldRange(NxmFieldCtMark)
-	a.load(field, uint64(mark.value), mark.field.rng)
+func (a *ofCTAction) LoadToCtMark(marks ...*CtMark) CTAction {
+	for _, mark := range marks {
+		field, _, _ := getFieldRange(NxmFieldCtMark)
```

Comment:
Does it need to create a new "field" for every mark?

Reply:
Good catch, it doesn't need to.